spec/unit/blockscore/person_spec.rb

```diff
@@ -3,6 +3,23 @@ module BlockScore
     let(:api_stub) { @api_stub }
     let(:action) { -> { create(:person).details } }
 
+    context 'when person has existing question sets' do
+      let(:person) { create(:person, question_set_count: 1) }
+      subject(:question_sets) { person.question_sets }
+      its(:size) { should eql(1) }
+      its(:data) { should be(person.attributes[:question_sets]) }
+      its(:first) { should be_an_instance_of(QuestionSet) }
+
+      it 'should load attributes' do
+        expect(question_sets.first.attributes).not_to be(nil)
+      end
+    end
+
+    it "#id" do
```

Comment:
Style/StringLiterals: Prefer single-quoted strings when you don't need string interpolation or special symbols.